presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestSwapAdjacentWindowsBySpecifications.java

```diff
@@ -59,9 +59,11 @@ public TestSwapAdjacentWindowsBySpecifications()
                 RANGE,
                 UNBOUNDED_PRECEDING,
                 Optional.empty(),
+                Optional.empty(),
```

Comment:
Also changed in ported PR, which does not have the additiional tests https://github.com/trinodb/trino/pull/5639/files#diff-fdca9a1d87656c730dc2f24162b6328b441f5bace128058c4f4b6d3cd5a9b412